Makefile.am

```diff
@@ -183,6 +185,14 @@ release:
 	$(MAKE) po-pull
 	$(MAKE) dist
 
+container-release:
+	$(CONTAINER_ENGINE) run \
+	$(CONTAINER_TEST_ARGS) \
+	$(CI_TEST_ARGS) \
+	$(CONTAINER_ADD_ARGS) \
+	$(RELEASE_NAME):$(CI_TAG) \
+	sh -exc './autogen.sh && ./configure && make release'
```

Comment:
Could we use `run-build-and-arg`, please? Then we wouldn't have to clean up the repository. Or is it a different use case?

Reply:
I don't think we want that. We need to have tarball as output and I don't really want to move the tarball around in the container to get it out. Also you should not be bothered by messing up the repository because release should always be from a new checkout as the guides said.